pkgs/applications/science/physics/elmerfem/default.nix

```diff
@@ -13,19 +13,45 @@ stdenv.mkDerivation rec {
 
   hardeningDisable = [ "format" ];
 
-  nativeBuildInputs = [ cmake gfortran pkg-config git ];
-  buildInputs = [ mpi blas liblapack qt4 qwt6_qt4 ];
+  nativeBuildInputs = [
+    cmake
+    gfortran
+    pkg-config
+    git
```

Comment:
Why do we need git?

Reply:
We probably don’t, looks like an oversight from when I originally wrote this.

Reply:
done